x-pack/plugins/dataset_quality/common/data_streams_stats/data_stream_details.ts

```diff
@@ -7,4 +7,5 @@
 
 export interface DataStreamDetails {
   createdOn?: number;
+  lastActivity?: number;
 }
```

Comment:
**nit**: this interface seems to represent the same data structure derived [here](https://github.com/elastic/kibana/pull/174906/files#diff-e94324a78c9896f654ad0dbf7d7291b22a0d93d336bc23a5ba534841f16cce5cR62) from the rt type, can we keep a single source of truth?

Reply:
Addressed in [bea5074](https://github.com/elastic/kibana/pull/174906/commits/bea50744b91d3e693d89d7602a0a0cf57ccb8f0e).